packages/react-components/react-teaching-popover-preview/docs/Spec.md

```diff
@@ -0,0 +1,194 @@
+# @fluentui/react-teaching-popover-preview Spec
+
+## Background
+
+A Teaching Popover is a structured popover to showcase information about a new component feature to a user. It should be attached via a trigger to a button, info tip, or component - or for further extension, callout(s). TeachingPopover can also be displayed programmatically, in this case it's intent should be announced to the user on launch to define context for accessibility users.
+
+For a simple feature, a single paged TeachingPopover can be used to display core information, while extensive reading can be linked via a 'Learn More' secondary action.
+
+For more complicated features, we recommend using the TeachingPopoverCarousel, this will enable multiple steps of information with an associating title/image, and can guide the user through multi-step tutorials - in situations where a single page is likely to require multiple TeachingPopovers pointing to various components, the upcoming TeachingCallout component will be better suited.
+
+## Prior Art
+
+- [OpenUI Research - Popover](https://open-ui.org/components/popup.research/)
+- [OpenUI Research - Carousel](https://open-ui.org/components/carousel.research/)
+- [Fluent V9 - Task: TeachingPopover](https://github.com/orgs/microsoft/projects/786/views/1?pane=issue&itemId=24403213)
+
+### v0/v8 components
+
+- [v8 TeachingPopover](https://developer.microsoft.com/en-us/fluentui#/controls/web/TeachingPopover)
+
+## Sample Code
```

Comment:
Usually, we try to keep the spec in a separate PR so we can review without all the other files and have a longer conversation. If this is just an initial draft, then it's okay :)